### ISSUE TITLE
Cannot find module 'react/lib/ReactNativePropRegistry' from computeProps.js

### ISSUE THREAD

**Issue by msom** [GeekyAnts/react-native-easy-grid#18]
with react `15.4.1`

`var ReactNativePropRegistry = require('ReactNativePropRegistry');` seems to work

**Comment by breezykermo** [comment]
I was running into the same issue, downgrading to React 15.3.x seems an initial quick fix.

**Comment by ujwal-setlur** [comment]
I have issued a pull request for this.

**Comment by dvolland** [comment]
Yes. Version 0.1.7 is not working. Reverting to 0.1.6 for now.

**Comment by MehdiZonjy** [comment]
changing it to the following does the trick on my end
```javascript
import ReactNativePropRegistry from 'react/lib/ReactNativePropRegistry';
```

**Comment by SupriyaKalghatgi** [comment]
@msom `var ReactNativePropRegistry = require('ReactNativePropRegistry');` dd'nt work for me

**Comment by SupriyaKalghatgi** [comment]
@dvolland Version 0.1.7 worked for me with
```
"react": "15.4.1",
"react-native": "0.38.0",
```

**Comment by SupriyaKalghatgi** [comment]
Fixed with latest release [0.1.7](https://github.com/GeekyAnts/react-native-easy-grid/releases/tag/v0.1.7)

**Comment by kiranruth** [comment]
I'm getting the same bug in 0.1.8 .
